src/platform_impl/linux/x11/mod.rs

```diff
@@ -57,8 +62,7 @@ use crate::{
 };
 
 const X_TOKEN: Token = Token(0);
-const USER_TOKEN: Token = Token(1);
-const REDRAW_TOKEN: Token = Token(2);
+const USER_READRAW_TOKEN: Token = Token(1);
```

Comment:
Shouldn't this be called `USER_REDRAW_TOKEN`?

Reply:
Haha, yes absolutely :smile:.